### PR TITLE
Added square_error_cost

### LINKED ISSUE #17425: square_error_cost

### PR DESCRIPTION
**Pull request by m-gopichand**
Close #17425

### CODE REVIEW

**Comment by mobley-trent** [comment]
Hey @m-gopichand 👋

File: ivy/functional/frontends/paddle/nn/functional/loss.py [hunk 1]
```diff
@@ -113,3 +113,9 @@ def log_loss(input, label, epsilon=0.0001, name=None):
         (1 - label) * ivy.log(1 - input + epsilon)
     )
     return out
+
+
+@with_supported_dtypes({"2.4.2 and below": ("float32", "float64")}, "paddle")
```
Comment (mobley-trent) [inline ivy/functional/frontends/paddle/nn/functional/loss.py:118]: Latest paddle version is `2.5.0`

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_paddle_loss.py [hunk 2]
```diff
@@ -196,3 +196,54 @@ def test_paddle_log_loss(
...
+    input_dtype, pred = dtype_and_pred
+    target_dtype, target = dtype_and_target
+    helpers.test_frontend_function(
+        input_dtypes=[input_dtype, target_dtype],
+        frontend=frontend,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        pred=pred,
+        target=target,
```
Comment (mobley-trent) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_paddle_loss.py:246]: The function doesn't have `pred` or `target` as inputs. Simple reformat to:

```
input=pred
label=target
```

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_paddle_loss.py [hunk 3]
```diff
@@ -196,3 +196,54 @@ def test_paddle_log_loss(
...
+        min_value=-1,
+        max_value=1,
+        exclude_min=True,
+        exclude_max=True,
+        min_num_dims=1,
+        max_num_dims=1,
+        min_dim_size=2,
+        shape=(5,),
+    ),
+    reduction=st.sampled_from(["none", "sum", "mean"]),
```
Comment (mobley-trent) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_paddle_loss.py:226]: This function doesn't have a reduction argument.

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_paddle_loss.py [hunk 4]
```diff
@@ -196,3 +196,54 @@ def test_paddle_log_loss(
...
+    reduction,
+    fn_tree,
+    test_flags,
+    frontend,
+    on_device,
+):
+    input_dtype, pred = dtype_and_pred
+    target_dtype, target = dtype_and_target
+    helpers.test_frontend_function(
+        input_dtypes=[input_dtype, target_dtype],
```
Comment (mobley-trent) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_paddle_loss.py:240]: Only one input dtype is necessary, choose any one of the two.

**Review by mobley-trent — CHANGES_REQUESTED** [review]

**Comment by mobley-trent** [comment]
Hey @m-gopichand I've left a few pointers for you to improve this PR

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Pull request closed without merging**